inputs/templates/test/RenameVcfSamples/RenameVcfSamples.json.tmpl

```diff
@@ -0,0 +1,321 @@
+{
+  "RenameVcfSamples.vcf": {{ test_batch.clean_vcf | tojson }},
+  "RenameVcfSamples.prefix": {{ test_batch.name | tojson }},
+  "RenameVcfSamples.current_sample_ids": [
```

Comment:
Just thinking about this again. In order to be consistent with other templates we should use `{{ test_batch.samples }}` here and create a new `{{ test_batch.alternate_ids }}` or the like for the new ids.